app/src/main/java/com/example/soswedding/ui/forgotPassword/ForgotPasswordFragment.java

```diff
@@ -79,11 +96,31 @@ public void onCancelClicked(){
                 .commit();
 
     }
-    public void onResetClicked(View view){
-        //TODO: implementation to come
 
+    public void onResetClicked(View view){
+        if(emailInput.length()>0){
+            verifyEmail(emailInput.getText().toString());
+        }else {
+            popUp("Fail to send reset password email!");
+        }
     }
 
+    private void popUp(String message){
+        Toast t = Toast.makeText(getActivity().getApplicationContext(),message,Toast.LENGTH_SHORT);
+        t.show();
+    }
 
-
+    private void verifyEmail(String email){
```

Comment:
Did you test this on your phone & your email?

Reply:
yes  I did